shared/wallets/transaction-details/index.tsx

```diff
@@ -287,8 +296,20 @@ export const TimestampLine = (props: TimestampLineProps) => {
   )
 }
 
+const ConvertedCurrencyLabel = (props: ConvertedCurrencyLabelProps) => (
+  <Kb.Box2 direction="vertical" noShrink={true}>
+    <Kb.Text type="BodyBigExtrabold">
+      {props.amount} {props.assetCode || 'XLM'}
+    </Kb.Text>
+    <Kb.Text type="BodySmall">/{props.issuerDescription || 'Unknown issuer'}</Kb.Text>
```

Comment:
cc @patrickxb on if we need this `Unknown issuer` check or if that'll always be provided server-side.

Reply:
It's good to be resilient like this. The payment field plumbing tends to break or change over time.

Reply:
Yes, as discussed in chat, this sounds good to me.